ruby/lib/minitest/queue/runner.rb

```diff
@@ -21,6 +21,7 @@ def self.invoke(argv)
 
       def initialize(argv)
         @queue_config = CI::Queue::Configuration.from_env(ENV)
+        Minitest.seed = @queue_config.seed.to_i
```

Comment:
Since this was added in the latest version, we want to check if the method exists.

```suggestion
        if Minitest.respond_to?(:seed=)
          Minitest.seed = @queue_config.seed.to_i
        end
```

Reply:
Also `parse()` takes a `--seed` option, so you want to do that after `parse`.

Reply:
both very correct, ty